### ISSUE TITLE
SAP showing solidarity - Closing all businesses

### ISSUE THREAD

**Issue by cyb3rko** [vshymanskyy/StandWithUkraine#26]
SAP has closed all their businesses in Russia and will ignore the ~468.000.000 Euros they made in Russia every year.

I hope that's enough to get SAP Archivelink off this list.

**Comment by vshymanskyy** [comment]
Thanks! Please unclude a link to the source, if possible. I'm goin to remove SAP, ASAP :)

**Comment by vshymanskyy** [comment]
@cyb3rko ?

**Comment by cyb3rko** [comment]
Ok, I just saw the numbers differ. I updated it to "only" 468 million, as this is the minimum the company loses in a year now.

One of the many sources is this page:
https://www.computerweekly.com/news/252514133/SAP-and-Oracle-express-solidarity-with-Ukraine-against-Russia

**Important cite:**
"Economic sanctions against Russia are an important mechanism in the efforts to restore peace,” said Klein. “We are in constant exchange with governments around the world, have every confidence in their guidance, and fully support the actions taken so far. **We are stopping business in Russia aligned with sanctions and, in addition, pausing all sales of SAP services and products in Russia.**”
